### PR TITLE
Drop noetic 3.14

### PR DESCRIPTION
**Pull request by f-fl0**

### CODE REVIEW

**Comment by at-wat** [comment]
Please make PRs to https://github.com/seqsense/aports-ros-experimental and https://github.com/seqsense/aports-ros-updater as well.
Let's drop 3.14 once everything are ready.

File: README.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ Docker image of ROS (Robot Operating System) on Alpine Linux
 ROS2 humble is now available for Alpine 3.17 as `humble-3.17`.
 
 ### April 2023
-`melodic` (Melodic EOL on May 2023) and `noetic-3.11` (Alpine 3.11 EOL on May 2020) are dropped.
-These images are still available but no longer updated. It is highly recommended to update to `noetic-3.14` or `noetic-3.17`.
+`kinetic` (Kinetic EOL on May 2021), `melodic` (Melodic EOL on June 2023), `noetic-3.11` (Alpine 3.11 EOL on November 2021) and `noetic-3.14` (Alpine 3.14 EOL on May 2023) are dropped.
```
Comment (at-wat) [inline README.md]: This is past news and `### January 2024` should be added

Comment (f-fl0) [reply README.md]: Fixed in 2a4afe896814966db79699e0545db2f7e60a8f5a. I left the changes to the `April 2023` section to mention `kinetic` but I can revert them if needed.

File: README.md [hunk 2]
```diff
@@ -3,11 +3,15 @@ Docker image of ROS (Robot Operating System) on Alpine Linux
...
+### January 2024
+`noetic-3.14` (Alpine 3.14 EOL on May 2023) are dropped. These corresponding images are still available but no longer updated.
+It is highly recommended to update to `noetic-3.17`.
+
 ### May 2023
 ROS2 humble is now available for Alpine 3.17 as `humble-3.17`.
 
 ### April 2023
-`melodic` (Melodic EOL on May 2023) and `noetic-3.11` (Alpine 3.11 EOL on May 2020) are dropped.
+`kinetic` (Kinetic EOL on May 2021), `melodic` (Melodic EOL on June 2023) and `noetic-3.11` (Alpine 3.11 EOL on November 2021) are dropped.
```
Comment (at-wat) [inline README.md]: We dropped kinetic build on 2021
https://github.com/alpine-ros/alpine-ros/pull/18

Comment (f-fl0) [reply README.md]: I just reverted the changes.

**Review by at-wat — APPROVED** [review]
LGTM

**Pull request merged**